bibliopixel/main/update.py

```diff
@@ -0,0 +1,29 @@
+"""
+Update BiblioPixel's dependencies.
+"""
+
+import os, sys, subprocess
+
+
+def run(args, settings):
+    root = os.path.dirname(os.path.dirname(os.path.dirname(__file__)))
+    code = 0
+
+    with open(os.path.join(root, 'requirements.txt')) as f:
```

Comment:
Just a note that this only works because we include requirements.txt in MANIFEST.in
But please test this by running `python setup.py sdist`, then pip installing the generated tar.gz file in a venv. Only way to make sure this will actually work when uploaded to pypi.